spec/spec_helper.rb

```diff
@@ -1,4 +1,12 @@
 require "rspec/json_expectations"
+
+require 'webmock/rspec'
+include WebMock::API
+# By default, allow webmock to connect.  In a specific
+# test to use, disallow webmock, stub_request, and then
+# allow_net_connect! in the test.
+WebMock.allow_net_connect!
```

Comment:
I think `WebMock` was included before due to VCR, so I'm not sure if this `allow_net_connect!` is needed?  And if it isn't, then we don't need to disable and re-enable it in the specs maybe?